pages/dkp/kommander/2.2/insights/insights-setup/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle: Insights setup and configuration
+title: Insights setup and configuration
+excerpt: Enable DKP Insights in managmenet and attached/managed clusters
+menuWeight: 20
+beta: false
+enterprise: false
+techPreview: true
+---
+
+DKP Insights is installed by default on your Management Cluster, and then you need to manually enable the DKP Insights Engine in order to begin viewing insight items for your environment.
```

Comment:
I think we need to make it clear that only the UI is installed 'by default'.   Also,  do we have a preferred phraseology/capitalization for "Insights items"?  "Insights Items"?   

```suggestion
The DKP Insights UI is installed by default on your Management Cluster.  To begin viewing Insight items for your environment, you need to manually enable the DKP Insights Engine.
```

Reply:
@nataliepina does the empty state of Insights give the user the heads up that they need to do this?

Reply:
It does not. It only states that they are no insights available and links to the docs.